src/background.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
```

Comment:
Nit, `meta` is a self-closing tag in HTML5 and so we could omit the closing `/` and just write it as `<meta charset="utf-8">`.

Reply:
Prettier gives an error on this. I see an issue already filed [here](https://github.com/prettier/prettier/issues/5246) regarding this.
So, I have asked prettier to ignore this for now.